src/app/Main/Body/QueryPage/Services/ContainerServices/getConditionalFilters.service.tsx

```diff
@@ -0,0 +1,57 @@
+import _ from "lodash";
+import { FilterInterface } from "../../../../../Shared/Model/Filter.model";
+import { ApiResponse, callApiService } from "../../../../../Core/callApi.service";
+import { FilterConfigDTO } from "../../../../../Shared/Model/Api_Filter.model";
+import { FILTER_URL } from "../../../../../Shared/URLs";
+import { generateUniqueValuesService } from "../generateUniqueValues.service";
+
+export async function getConditionalFilters(props: {

```

Comment:
Should be called fetchConditionalFilters, so the caller does know that the api is called here.

Reply:
Done.